handler.go

```diff
@@ -269,7 +310,11 @@ var (
 	{{- if .IsDev }}
 		{{ .PluginReactPreamble }}
 		<script type="module" src="{{ .ViteURL }}/@vite/client"></script>
-		<script type="module" src="{{ .ViteURL }}/src/main.tsx"></script>
+		{{ if ne .ViteEntry "" }}
```

Comment:
Maybe replace this with ...

```suggestion
		{{- if ne .ViteEntry "" }}
```

?

Reply:
Yes. Will do.